sandbox/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+mod account;
+pub mod extensions;
+pub mod helpers;
+pub mod tx;
+
+pub use account::{Account, SigningAccount};
+pub use helpers::*;
+
+pub use anyhow;
+pub use near_api as api;
+// NOTE: that is not ok - errors should be separated to another mod
+pub use near_openapi_types as openapi_types;
+pub use near_sandbox as sandbox;
+
+use near_api::{NetworkConfig, RPCEndpoint};
+use near_sandbox::GenesisAccount;
+use near_sdk::{AccountId, NearToken};
+
+use crate::extensions::account::ParentAccountExt;
+
+const INITIAL_ACCOUNT_BALANCE: NearToken = NearToken::from_near(10);
+pub struct Sandbox {
+    root: SigningAccount,
+
+    sandbox: near_sandbox::Sandbox,
```

Comment:
Wdyt about making it shared:
```suggestion
    sandbox: Arc<near_sandbox::Sandbox>,
```
And then having this method:
```rust
impl Sandbox {
    pub fn sub_sandbox(&self, name: impl AsRef<str>, amount: NearToken) -> Self {
        Self {
            root: self.root().create_subaccount(name, amount),
            sandbox: self.sandbox.clone(),
        }
    }
}
```
And then we can have:
```rust
pub static SHARED_SANDBOX: LazyLock<Sandbox> = LazyLock::new(|| Sandbox::new("test"));
```
(there is a caveat with async LazyLock)